WordPress/gutenberg
# Storybook: Update control types from `null` to `undefined`

## What?
This PR updates `null` control types to `undefined`.

Part of #67574

## Why?
Preparatory work for upgrading Storybook v8 - `null` controls cause type errors with v8.

## How?
Just updating the types.

## Testing Instructions
Verify Storybook builds well and test a few controls to ensure they work the same way.

### Testing Instructions for Keyboard
Same

## Screenshots or screencast 
None
